src/admin-portal/pages/campaign/campaign-statistics/campaign-statistics.js

```diff
@@ -37,36 +37,50 @@ export function CampaignStatistics({}) {
   const { blow, pop } = useBubblyBalloons();
 
   const {
-    data: campaign,
-    isLoading: campaignLoading,
-    isValidating: campaignValidating,
-    error: campaignError,
+    data: stats,
+    isLoading: statsLoading,
+    isValidating: statsValidating,
+    error: statsError,
   } = useSWR(
-    `campaign.info/${id}`,
+    `campaign.stats.get-${id}`,
     async () => {
-      return await fetchCampaign(id);
+      return await fetchCampaignStats(id);
     },
     {
       onSuccess: (data) => {},
     },
   );
 
-  const CAMPAIGN = campaign || {};
 
-  const statistics = Object.entries(CAMPAIGN?.stats || {});
+
+  if(!statsLoading && statsError){
+    return <div className="text-center mt-5">
+      <h3 className="text-danger">An error occurred</h3>
+    </div>
+  }
+
+  if(statsLoading || statsValidating){
```

Comment:
Pls remove the `|| statsValidating` from this boolean expression cos it will cause the Loader to show even when there's data and swr is making a request for to check if the data is stale
Just check check `statsLoading` as that is only updated once